app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp, file, file_copy = command.split()
+    if cp != "cp" or file == file_copy or len(command.split()) != 3:
```

Comment:
It's better to perform command length check before you split the command into variables.